Studio-Lovelies/GG-JointJustice-Unity
# Implemented ability to toggle visibility of actors via SHOW_ACTOR action

## Summary
Adds `&SHOW_ACTOR:true` and `&SHOW_ACTOR:false` actions to toggle visibility of the currently displayed actor.

The placeholder for the logic to toggle actor visibility was in `SceneController`, but as it's tied to manipulating the sprites and things like playing emotes or talking are already inside `ActorController`, I've moved the methods there. Open for feedback on that however, if you feel it makes sense in another place!

## Before/after screenshots and/or animated gif

## Testing instructions
1. Open the `Inky- TestScene`
2. Change the `StoryController` to run `/Assets/Resources/InkDialogueScripts/TestScripts/VisibilityTest`
3. Hit play and play through the script and notice Actors appearing and disappearing

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - Closes #127
